keep/api/core/db.py

```diff
@@ -1992,6 +1992,7 @@ def get_last_incidents(
         query = (
             session.query(
                 Incident,
+                subquery.c.last_updated_time
```

Comment:
Also, please take a look at https://github.com/keephq/keep/pull/1613 - I've optimized this behaviour and you can use last_seen_time field in Incident

Reply:
Looks convenient, thanks! It makes sense to update once your PR is merged, right?